src/com/dtmilano/android/viewclient.py

```diff
@@ -2604,7 +2604,13 @@ def __obtainDeviceSerialNumber(device):
             # If there's only one device connected get its serialno
             adb = ViewClient.__obtainAdbPath()
             if DEBUG_DEVICE: print >>sys.stderr, "    using adb=%s" % adb
-            s = subprocess.Popen([adb, 'get-serialno'], stdout=subprocess.PIPE, stderr=subprocess.PIPE, env={}).communicate()[0][:-1]
```

Comment:
Were you able to reproduce this from the command line, using `adb get-serialno` and setting the environment?